apps/insights/src/components/MobileMenu/mobile-menu.module.scss

```diff
@@ -0,0 +1,53 @@
+@use "@pythnetwork/component-library/theme";
+
+.mobileMenuTrigger {
+    display: none;
+    @include theme.mobile {
+        display: block;
+    }
+}
+
+.mobileMenuOverlay {
+    background: rgba(0,0,0,0.4);
```

Comment:
`rgba` is effectively deprecated and should be replaced with `rgb` instead: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb

So this would be:

```scss
background: rgb(0 0 0 / 40%);
```

or (I prefer this):

```scss
background: rgb(from black r g b /40%)
```